### PR TITLE
Translation management with Crowdin

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

This PR enables automatic translation management using [Crowdin](https://crowdin.com).

**_TL;DR:_** Developers now only need to update English strings. Rest is handled via Crowdin and its GitHub integration.

**_IMPORTANT:_**  From now on, updates to the non English locale files should only be done from the Crowdin UI!

#### New notes added to the mobile README

We are using [Crowdin](https://clabs.crowdin.com/) to manage the translation of all user facing strings in the app.

During development, developers should only update the main language files in English (en-US).

The `main` branch of this repository is automatically synced with our Crowdin project. Source files in Crowdin are updated automatically and ready translations are pushed as a pull request.

Translation process overview:

1. Developers update the English strings in the branch they are working on.
1. When the corresponding PR is merged into `main`, Crowdin integration automatically picks up changes to the English strings.
1. Crowdin then auto translates the new strings and opens a PR with them from the `l10n/main` branch
1. We can then manually check and edit the translated strings in the Crowdin UI. The changes will be reflected in the PR after 10 mins.
1. When we are happy with the changes, we can merge the PR and delete the related `l10n/main` branch to avoid possible future conflicts. Once new translations are made in Crowdin, a new `l10n/main` branch will be automatically created again.

When making a release, we should make sure there are no outstanding translation changes not yet merged into `main`.
i.e. no Crowdin PR open and the translation status for all supported languages is at 100% and approved on Crowdin.

#### Why Crowdin?

There are other similar tools that @aleksey-nov and I evaluated.

Crowdin matched all our criteria:
- great editing UI
- great tooling (GitHub integration, CLI, API, editors, etc)
- supports a wide range of file formats
- free for Open Source projects
- custom workflows
- possibility of crowdsourcing translations
- Over-the-Air updates with a CDN
- support for plurals
- translation QA
- adding screenshots as context to help translators
- and more ;)

#### Future improvements:

Crowdin also supports Over-the-Air updates via a CDN.
This would require a small client side change so it downloads the json files at the appropriate frequency.
The main advantage is that it would allow cutting new releases without waiting for all new strings to be fully translated.
And also updating copies on versions of the app that are already released.

**Note:** You'll see that some Spanish and Portuguese files were touched in the PR, but they are only reorganization of the strings so it matches the order from the English files.

### Tested

Tested with this branch.

### Backwards compatibility

Yes

### CODE REVIEW

**Review by etuleu — APPROVED** [review]
Really nice idea to use branches!

**Review by gnardini — APPROVED** [review]
Awesome not to have to manually do this anymore 🙌

**Comment by Celoqa** [comment]
Hey @jeanregisser Can you please provide credentials for Crowdin so that we can manually check for translation. And also we can perform Localization testing for our three app languages.

**Comment by Celoqa** [comment]
Hey @jeanregisser Let me please know any updates for above comment.
Thanks..!!

**Comment by gnardini** [comment]
I'm asking if it's necessary to test this and will get back to you.

**Comment by gnardini** [comment]
No need to check on Crowdin, all translations should show up on GitHub here: https://github.com/celo-org/wallet/tree/main/packages/mobile/locales
